core/domain/platform_parameter_domain.py

```diff
@@ -385,6 +397,23 @@ def _is_first_version_smaller(self, version_a, version_b):
                 return False
         return False
 
+    def _match_version_flavor(self, flavor, client_version):
+        """Matches the client version flavor.
+
+        Args:
+            flavor: str. The flavor to match, e.g. 'alpha', 'beta', 'test',
+                'n/a'.
+            client_version: str. The version of the client, given in the form
+                of '<version>-<hash>-<flavor>'. The hash and flavor of client
+                version is optional, if absent, the flavor is considered 'n/a'.
```

Comment:
Shouldn't the default be 'release' not 'n/a'? Though per my comment above, I suppose 'unknown' may be a more appropriate default if we now expect the flavor to be present.

Reply:
Done.